index.bs

```diff
@@ -284,6 +284,22 @@ Restrictions and Clarifications {#restrictions}
       function and user gestures.
     </note>
 
+8. If an <a
+    href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element">iframe</a>
+    element is [=skipped=], then the content of the frame does not paint or
+    participate in hit testing. The user-agent should skip <a
+    href="https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering">Update
+    the Rendering</a> step of the Processing Model in the frame, if possible.
```

Comment:
```suggestion
    the Rendering</a> step of the Processing Model in the frame.
```

Reply:
Also, s/frame/frame's window's event loop/

Reply:
Re "if possible": I see what you mean now, based on the note below...

Reply:
I reworded this a bit. frame's window's event loop is extremely confusing to parse... I said frame's event loop, which I think captures sufficient information. WDYT?

Also added "see note below" after "if possible"